tests/conformance_jekyll/slugify_test.rs

```diff
@@ -0,0 +1,49 @@
+use liquid;
+
+#[test]
+fn test_slugify_default() {
```

Comment:
Please make this file a one-to-one mapping with Jekyll's tests

if you want to make your own tests, feel free to put them in your `jekyll.rs` file